web/modules/custom/joinup_core/joinup_core.behat.inc

```diff
@@ -79,6 +89,26 @@ class JoinupCoreSubContext extends DrupalSubContextBase {
     $button->press();
   }
 
+  /**
+   * Counts the number of chips on the page.
+   *
+   * @return int
+   *   The number of chips on the page.
+   */
+  protected function countChips(): int {
+    return count($this->getChips());
```

Comment:
Why split off such a simple functionality? And it is also used just once? It does not really help with the reduction of complexity, nor it would cause a problem to the readability of the code.

Reply:
Oh I always split off the logic in a separate protected method so it can be reused and the step definitions are just handling the arguments. But yeah this is a bit ridiculous :)

I'll remove it.